### ISSUE TITLE
GBIF changing verbatim higher taxonomy

### ISSUE THREAD

**Issue by jhnwllr** [gbif/portal-feedback#3090]
I have been doing some research into GBIF **verbatim** higher taxonomy changes. 

Occasionally GBIF will alter the verbatim (publisher provided) higher taxonomy. 

GBIF will do this even if it is a **big** change like **insects** -> **plants**  

Sometimes this is the **right** decision (insect -> plant)
https://www.gbif.org/occurrence/2814576412

but other times it is **wrong**. (fish -> butterfly)
https://www.gbif.org/occurrence/1322148632

I decided to check all obviously wrong pairings like... 

fish -> bird
insect -> flower
jelly fish -> reptile
mammal -> butterfly 

And many other obviously wrong pairings...

Within **obviously wrong pairings**, GBIF makes the right decision around **50%** of time.  But it also makes the **wrong** decision 50% of the time.

Since in the **obviously right scenarios** (flower -> another type of flower ect.), GBIF’s success rate is much higher, changing higher verbatim taxonomy is **vastly a net good**. 

Should we perhaps consider **flagging** obviously wrong changes somehow? 

I think it is a hard problem. I am just leaving my work here for future reference and checking with future backbone updates.   

Below I list some manually reviewed cases of changes.  

**Good Changes**: 
https://www.gbif.org/occurrence/683893442
https://www.gbif.org/occurrence/683904634
https://www.gbif.org/occurrence/1424788186
https://www.gbif.org/occurrence/2621909129
https://www.gbif.org/occurrence/2621942661
https://www.gbif.org/occurrence/2621928602
https://www.gbif.org/occurrence/2621913849
https://www.gbif.org/occurrence/889485810
https://www.gbif.org/occurrence/1147096293
https://www.gbif.org/occurrence/2313475033
https://www.gbif.org/occurrence/2313475058
https://www.gbif.org/occurrence/2621924422
https://www.gbif.org/occurrence/2621930943
https://www.gbif.org/occurrence/2621939603
https://www.gbif.org/occurrence/2621926161
https://www.gbif.org/occurrence/232973486
https://www.gbif.org/occurrence/2621915165
https://www.gbif.org/occurrence/1502236880
https://www.gbif.org/occurrence/1416926154
https://www.gbif.org/occurrence/232673506
https://www.gbif.org/occurrence/232715997
https://www.gbif.org/occurrence/889485765
https://www.gbif.org/occurrence/233021782
https://www.gbif.org/occurrence/233021978
https://www.gbif.org/occurrence/2814585368
https://www.gbif.org/occurrence/234975210
https://www.gbif.org/occurrence/234975324
https://www.gbif.org/occurrence/234975195
https://www.gbif.org/occurrence/234975326
https://www.gbif.org/occurrence/2013910269
https://www.gbif.org/occurrence/1416926154
https://www.gbif.org/occurrence/232673549
https://www.gbif.org/occurrence/232716321
https://www.gbif.org/occurrence/1848313373
https://www.gbif.org/occurrence/1950909039
https://www.gbif.org/occurrence/2274772765
https://www.gbif.org/occurrence/1950909112
https://www.gbif.org/occurrence/1438010409
https://www.gbif.org/occurrence/2814576412

**Bad Changes**: 
https://www.gbif.org/occurrence/1654939545
https://www.gbif.org/occurrence/1265270165
https://www.gbif.org/occurrence/1322148632
https://www.gbif.org/occurrence/1317780441
https://www.gbif.org/occurrence/1322476397
https://www.gbif.org/occurrence/1291371671
https://www.gbif.org/occurrence/1291525117
https://www.gbif.org/occurrence/1291413917
https://www.gbif.org/occurrence/1944649992
https://www.gbif.org/occurrence/2823481620
https://www.gbif.org/occurrence/1022842454
https://www.gbif.org/occurrence/2418067199
https://www.gbif.org/occurrence/2572309964
https://www.gbif.org/occurrence/1449740481
https://www.gbif.org/occurrence/1842100054
https://www.gbif.org/occurrence/2234219777
https://www.gbif.org/occurrence/1919782638
https://www.gbif.org/occurrence/2630034535
https://www.gbif.org/occurrence/2572309964
https://www.gbif.org/occurrence/1449740481
https://www.gbif.org/occurrence/1300095503
https://www.gbif.org/occurrence/2234225909
https://www.gbif.org/occurrence/2579433555
https://www.gbif.org/occurrence/2864540422
https://www.gbif.org/occurrence/666576112
https://www.gbif.org/occurrence/666691336
https://www.gbif.org/occurrence/2234219708
https://www.gbif.org/occurrence/2234219791
https://www.gbif.org/occurrence/1981048046
https://www.gbif.org/occurrence/1268714724
https://www.gbif.org/occurrence/1147056421
https://www.gbif.org/occurrence/2274775279
https://www.gbif.org/occurrence/1211655089
https://www.gbif.org/occurrence/1054645042
https://www.gbif.org/occurrence/1269189252
https://www.gbif.org/occurrence/2239082462
https://www.gbif.org/occurrence/1234563692
https://www.gbif.org/occurrence/2235075664
https://www.gbif.org/occurrence/1438011942
https://www.gbif.org/occurrence/858542627
https://www.gbif.org/occurrence/2565027174
https://www.gbif.org/occurrence/232973486

**Comment by tucotuco** [comment]
If it is knowable that an interpretation is bad, isn't it also possible to
avoid doing it?

And a point of clarification. GBIF doesn't actually change verbatim data,
it interprets some of it and indexes the interpretation, leaving all
verbatim (as published anyway) data as is, correct?

**Comment by jhnwllr** [comment]
Yes thanks for the clarification. GBIF does not alter the verbatim it **interprets**. 

It is possible to avoid the bad ones, with updated checklists. I would hope that a **flag** could highlight needed checklists.  

In the end, since the higher taxonomy we get from publishers can be highly variable it might be hard to make a flag like this. I only considered verbatim higher taxa that matched exactly (letter-for-letter) something we already have in the backbone.

**Comment by thomasstjerne** [comment]
Thanks @jhnwllr this seems very useful

Basically there a two categories.

1. Misplaced taxa in the GBIF taxonomy.
2. The species matching sometimes give priority to name matches in cases where the higher classification provides information that should be given more weight.

Indeed the completeness and quality of the underlying checklists will solve issues in category 1. For example https://www.gbif.org/occurrence/2418067199 will be fixed as the [underlying taxon in COL](http://www.catalogue.life/data/taxon/3VYCL) has the right classification now. 

For category 2. it might be possible to adjust the matching. 
For example this Fish classification:
`Animalia > Chordata > Actinopterygii > Siluriformes > Ariidae > Sciades > Sciades leptaspis` 
is matched to this Beetle:
`Animalia  > Arthropoda > Insecta > Coleoptera > Cerambycidae > Sciades > Sciades leptaspis`

This is because the Fish species _Sciades leptaspis_ is missing in the GBIF backbone, but there is an exact match on the Beetle species name. Here the verbatim classification looks legit all the way from the root to the genus level and it would make sense to prioritise the higher classification and do a TAXON MATCH HIGHER RANK to the Fish genus _Sciades_.

But you also list examples where the relaxed approach to the verbatim classification makes sense.
For example this Bird has for some reason a Fish order in the middle of its verbatim classification:
`Animalia > Chordata > Actinopterygii > Passeriformes > Icteridae > Cacicus > Cacicus uropygialis`
which is matched to 
`Animalia > Chordata > Aves > Passeriformes > Icteridae > Cacicus > Cacicus uropygialis`
regardless of the wrong order. 
And similarly here, a Plant phylum and order is inserted in the middle of a Bat verbatim classification:
`Animalia > Tracheophyta > Magnoliopsida > Chiroptera > Vespertilionidae > Myotis > Myotis myotis` 
which is again correctly matched to:
`Animalia > Chordata > Mammalia > Chiroptera > Vespertilionidae > Myotis > Myotis myotis`

The two latter cases are quite obvious errors in the verbatim data, but this could also happen when e.g. genera are combined into new families and the verbatim data is not updated to reflect this.
It might be worth introducing a flag for occurrences where the the verbatim classification deviates from the taxon it is matched to.
